sympy/series/tests/test_series.py

```diff
@@ -385,3 +386,17 @@ def test_issue_23432():
 def test_issue_23727():
     res = series(sqrt(1 - x**2), x, 0.1)
     assert res.is_Add == True
+
+
+def test_issue_24266():
```

Comment:
You might want to look into `test_issue_14384` and change the last 3 tests to raise a `PoleError` rather than returning self.

Reply:
sure